experimental/packages/opentelemetry-sdk-metrics-base/src/exemplar/Exemplar.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { HrTime } from '@opentelemetry/api';
+import { ValueType, Attributes } from '@opentelemetry/api-metrics-wip';
+
+// A representation of an exemplar, which is a sample input measurement.
```

Comment:
From what I can recall, only block comments can be correctly recognized by doc tools as class and field descriptions. But I failed to find any resource on this :/.

```suggestion
/**
  * A representation of an exemplar, which is a sample input measurement.
```